pkg/ccl/backupccl/restore_job.go

```diff
@@ -706,6 +687,26 @@ func restore(
 	g.GoCtx(func(ctx context.Context) error {
 		ctx, progressSpan := tracing.ChildSpan(ctx, "progress-log")
 		defer progressSpan.Finish()
+		if !dataToRestore.isMainBundle() {
+			// We only update the progress for the primary data bundle (of which there
+			// can only be one).
+			return nil
+		}
+
+		progressLogger := jobs.NewChunkProgressLogger(job, len(importSpans), job.FractionCompleted(),
```

Comment:
took me a minute but I guess this is all threadsafe so it is okay to move it inside the goroutine. that said, if we wrapped the whole `GoCtx` in an `if`, it could sit inside the `if `but outside the `goroutine`. I donno if it matters though.

Reply:
Updated in the rebase.